storybook/components/DocsContainer/DefaultDocsContainer.tsx

```diff
@@ -53,6 +47,11 @@ export const DefaultDocsContainer = ({
   ...props
 }: DocsContainerProps & { children: React.ReactNode }): JSX.Element => (
   <DocsContainer {...props}>
-    <DocsLayout>{children}</DocsLayout>
+    <DocsLayout>
+      {/* <Title />
+      <KAIOInstallation exportNames="Tooltip" family="overlays" version="1" /> */}
+      <div className="test">{children}</div>
```

Comment:
I guess this is leftover from testing, right?

```suggestion
      {children}
```

Reply:
Ah yeah that can go for now.